web/src/components/pages/contribution/speak/speak.tsx

```diff
@@ -603,6 +631,50 @@ class SpeakPage extends React.Component<Props, State> {
       <>
         <div id="speak-page">
           {noNewClips && isLoading && <Spinner delayMs={500} />}
+          {isSubmitted && true && (
```

Comment:
I don't think this `true` is needed here. If yes then it should be assigned to a variable so it's clear what it means